js/src/components/paid-ads/campaign-assets-form.js

```diff
@@ -77,6 +79,14 @@ export default function CampaignAssetsForm( {
 
 	const [ baseAssetGroup, setBaseAssetGroup ] = useState( initialAssetGroup );
 	const [ hasImportedAssets, setHasImportedAssets ] = useState( false );
+	const [ countryCodes, setCountryCodes ] = useState( [] );
+	const { validateCampaignWithCountryCodes } =
+		useValidateCampaignWithCountryCodes( countryCodes );
+
+	const handleOnChange = ( _, values, arg ) => {
+		setCountryCodes( values.countryCodes );
+		onChange( _, values, arg );
```

Comment:
The plugin crashed after navigating to the campaign creating or editing page.

https://github.com/user-attachments/assets/39464a48-96ed-461d-babe-cf4fecee1e2a

Reply:
That should be fixed now @eason9487